clients/src/test/java/org/apache/kafka/clients/admin/KafkaAdminClientTest.java

```diff
@@ -514,13 +514,18 @@ public void testCreateTopicsRetryBackoff() throws Exception {
             TestUtils.waitForCondition(() -> ((KafkaAdminClient) env.adminClient()).numPendingCalls() == 1,
                 "Failed to add retry CreateTopics call");
 
-            time.sleep(retryBackoff);
+            long sleepTime1 = (long) (retryBackoff * Math.pow(2, 0) * 0.8) - 1;
+            long sleepTime2 = (long) (retryBackoff * Math.pow(2, 0) * 0.4) + 2;
 
+            time.sleep(sleepTime1);
+            assertEquals(1, ((KafkaAdminClient) env.adminClient()).numPendingCalls());
+
+            time.sleep(sleepTime2);
             future.get();
 
             long actualRetryBackoff = secondAttemptTime.get() - firstAttemptTime.get();
             assertEquals("CreateTopics retry did not await expected backoff",
```

Comment:
Instead of doing an `assertEquals` here, it might be more deterministic to check that the actual backoff fell between 0.8 and 1.2 times the configured retry backoff.

Reply:
As discussed offline, given the limitation of `MockTime`, what we have for the current test is fine for this.

Reply:
Yes. Let's keep it for now since the current test framework doesn't allow us to get the real backoff.